### PR TITLE
fix(DB/Tanaris): correct position of Jin'Zallah the Sandbringer

### PR DESCRIPTION
**Pull request by elthehablo**
## Changes Proposed:
-  move him to the hut
-  

## Issues Addressed:

- Progress https://github.com/azerothcore/azerothcore-wotlk/issues/16393
- Progress https://github.com/chromiecraft/chromiecraft/issues/1222

## SOURCE:

found from parsing a sniff supplied to me:

`Position: X: -7195.0405 Y: -2979.6877 Z: 9.052029
Orientation: 5.340707302093505859`

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
According to the creature's Wowhead page, it looks like they have multiple pooled spawn locations.

**Comment by elthehablo** [comment]
> According to the creature's Wowhead page, it looks like they have multiple pooled spawn locations.

😭

**Comment by elthehablo** [comment]
> According to the creature's Wowhead page, it looks like they have multiple pooled spawn locations.

Might be incorrect though. Seems like a lot of people talk about fighting him in the tent

**Comment by heyitsbench** [comment]
[Top comment](https://www.wowhead.com/wotlk/npc=8200#comments:id=2827825) references a point in a cluster that wouldn't apply to this PR's coordinates.

File: data/sql/updates/pending_db_world/update-jinzallah-spawn-posis.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--
+UPDATE `creature` SET `position_x` = -7195.0405, `position_y` = -2979.6877, `position_z` = 9.052029, `orientation` = 5.340707302093505859 WHERE `guid` = 51829 AND `id1` = 8200;
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/update-jinzallah-spawn-posis.sql:2]: Actually, you can add the VerifiedBuild as well 😛 
Just to differentiate that this particular spawn is sniffed

**Comment by heyitsbench** [comment]
I believe just the issue having the 'needs sniff' label is sufficient, don't think the PR needs it as well.

**Comment by elthehablo** [comment]
> I believe just the issue having the 'needs sniff' label is sufficient, don't think the PR needs it as well.

I need all my manpower to convince sniffers to do this for me

**Comment by Gultask** [comment]
Change 'Closes' to 'Progress' and feel free to wait but with a merged PR :happyge:

**Comment by elthehablo** [comment]
> Change 'Closes' to 'Progress' and feel free to wait but with a merged PR :happyge:

Done :happyge:

**Review by Gultask — APPROVED** [review]

**Pull request merged**